main.c

```diff
@@ -144,6 +145,8 @@ static void seat_handle_capabilities(void *data, struct wl_seat *seat,
 		struct wl_pointer *wl_pointer = wl_seat_get_pointer(seat);
 		create_pointer(state, wl_pointer);
 	}
+
+	wl_seat_destroy(seat);
```

Comment:
Oh, my bad. Maybe keep a reference of it in the state and destroy it on exit then?

Reply:
There is no guarantee that there will be only one seat

Reply:
A list then? Or do you just want to leave it for now? I can remove the line from my patch.

Reply:
Yeah, a list with all the burden that comes with it: adding a new struct and allocating list elements.

This can be done in another PR if you prefer.

Reply:
Splitting it into another PR sounds like a good idea to me.